### PR TITLE
HP-143 Provision staging and production on PDPH AWS subaccount

### PR DESCRIPTION
**Pull request by vkurup**
https://caktus.atlassian.net/browse/HP-143

The docs in `provision.md` explain what I did to get this set up, but they shouldn't need to be done again.

The docs in `deploy.md` explain tasks that every developer should be able to do successfully.

These changes have been deployed successfully to:
* prod: hip-prod.caktus-built.com
* staging: hip.caktus-built.com

### CODE REVIEW

File: docs/deploy.md [hunk 1]
```diff
@@ -176,12 +176,23 @@ to the cluster.
...
+
+Reference: https://github.com/caktus/caktus-hosting-services/blob/main/docs/aws-assumerole.md#aws-accounts
+
+### Useful Commands
 
 To see configured ansible variable like ``k8s_environment_variables`` you can use the following invoke command.
 
 ```sh
-(hip)$ inv staging info.get-ansible-vars
+(hip)$ inv staging info.print-ansible-vars
```
Comment (dchukhin) [inline docs/deploy.md:195]: Note: running this command complains about not having a region set up (`You must specify a region. You can also configure your region by running "aws configure".`) Adding the region for philly-hip in `~/.aws/credentials` fixed the issue.

**Review by dchukhin — APPROVED** [review]
Good job on this! 🎉

File: deploy/group_vars/all.yml [hunk 2]
```diff
@@ -5,22 +5,91 @@
...
+  stack_name: "{{ stack_name }}"
+  template_bucket: "aws-web-stacks-{{ app_name }}"
+  template_local_path: '{{ playbook_dir + "/stack/eks-no-nat.yml" }}'
+  create_changeset: true
+  termination_protection: true
+
+  template_parameters:
+    UseAES256Encryption: "true"
+    CustomerManagedCmkArn: ""
+    DomainName: "{{ app_name }}-prod.caktus-built.com"
```
[the remark this answers is not in the document]
Comment (vkurup) [reply deploy/group_vars/all.yml:49]: Kinda... it's one of those things that needs to be there for the stack to be created, but once it is created, we manage the domains in the django-k8s config.

File: docs/deploy.md (hunk 1 above)
Comment (vkurup) [reply docs/deploy.md:195]: Ahh, I have mine set in a separate file in `~/.aws/config` for some reason. I'll update the docs.

**Pull request merged**